### PR TITLE
Add docu for llamafile

### LINKED ISSUE #627: Add docu for llama file

### PR DESCRIPTION
**Pull request by JonasHelming**
fixed #627

### CODE REVIEW

File: src/docs/user_ai.md [hunk 1]
```diff
@@ -61,6 +62,29 @@ As an alternative to using an official OpenAI account, Theia IDE also supports a
...
+           "port": 30000 //you can choose a port to be used by llamafile
+       }
+   ]
+}
+```
+
+Replace "name", "uri", and "port" with your specific LlamaFile details.
+
+Managing LlamaFiles
+- Start a LlamaFile: Use the command "Start llamafile", then select the model you want to start.
```
Comment (sdirix) [inline src/docs/user_ai.md:84]:
```suggestion
- Start a LlamaFile: Use the command "Start Llamafile", then select the model you want to start.
```

File: src/docs/user_ai.md [hunk 2]
```diff
@@ -61,6 +62,29 @@ As an alternative to using an official OpenAI account, Theia IDE also supports a
...
+- Stop a LlamaFile: Use the "Stop llamafile" command.
```
Comment (sdirix) [inline src/docs/user_ai.md:85]:
```suggestion
- Stop a LlamaFile: Use the "Stop Llamafile" command, then select the running Llamafile which you want to terminate.
```

File: src/docs/user_ai.md [hunk 3]
```diff
@@ -61,6 +62,29 @@ As an alternative to using an official OpenAI account, Theia IDE also supports a
...
+           "uri": "file:///home/.../YourModel.llamafile",
+           "port": 30000 //you can choose a port to be used by llamafile
+       }
+   ]
+}
+```
+
+Replace "name", "uri", and "port" with your specific LlamaFile details.
+
+Managing LlamaFiles
```
Comment (sdirix) [inline src/docs/user_ai.md:83]:
```suggestion
The Theia IDE also offers convenience commands to start and stop your LlamaFiles:
```

File: src/docs/user_ai.md [hunk 4]
```diff
@@ -61,6 +62,29 @@ As an alternative to using an official OpenAI account, Theia IDE also supports a
...
+- Start a LlamaFile: Use the command "Start llamafile", then select the model you want to start.
+- Stop a LlamaFile: Use the "Stop llamafile" command.
+
+For more details on LlamaFiles, see the [LlamaFile documentation](https://github.com/Mozilla-Ocho/llamafile).
```
Comment (sdirix) [inline src/docs/user_ai.md:87]:
```suggestion
Please make sure that your LlamaFiles are executable.
For more details on LlamaFiles, including a quickstart, see the official [Mozilla LlamaFile documentation](https://github.com/Mozilla-Ocho/llamafile).
```

**Comment by JonasHelming** [comment]
We merge this after the release

**Review by sdirix — APPROVED** [review]

**Pull request merged**